PIL/BmpImagePlugin.py

```diff
@@ -56,131 +57,147 @@ def _accept(prefix):
     return prefix[:2] == b"BM"
 
 
-##
+#===============================================================================
 # Image plugin for the Windows BMP format.
-
+#===============================================================================
 class BmpImageFile(ImageFile.ImageFile):
-
-    format = "BMP"
+    """ Image plugin for the Windows Bitmap format (BMP) """
+    
+    #--------------------------------------------------------------- Description
```

Comment:
Oh yes, sorry, gonna fix it very soon. This is the comment style automatically used by Eclipse, and it did fit me until very recently, when I started using flake8.